docs/BasicCommunication/GetSystemInfo/assets/ccpu_versionChanged.txt

```diff
@@ -0,0 +1,37 @@
+participant App
+participant SDL
+participant HMI
+App->SDL:RAI
+note left of App:<align:center>RAI request processing\non hold until\nhmi_cooperating_ = true
+HMI->SDL:BC.OnReady
+SDL->SDL:ONHMIStartedCooperation
+note over SDL:<align:center>Do not set\nhmi_cooperating_= true\nyet
+SDL->HMI:BC.GetSystemInfo
+HMI-->SDL:BC.GetSystemInfo
+opt ccpu_version changed - Happy Path
+SDL->SDL:<align:center>Delete capabilities data\nfrom `HMICapabilitesCache` file
+loop for each interface
+SDL->HMI:[VR,UI,TTS...].IsReady
+HMI-->SDL:[VR,UI,TTS...]IsReady
```

Comment:
```suggestion
HMI-->SDL:[VR,UI,TTS...].IsReady
```
Sorry, seems I missed the missing dot here as well

Reply:
fixed fea4949593a3f6037ea88938fc99d8c8052b9036